src/Compilers/CSharp/Portable/Symbols/Source/ExplicitInterfaceHelpers.cs

```diff
@@ -17,21 +17,78 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
+#nullable enable
+    internal class ExplicitInterfaceMemberInfo(Binder binder, BindingDiagnosticBag diagnostics, ExplicitInterfaceSpecifierSyntax explicitInterfaceSpecifier, string name, string? aliasQualifier)
```

Comment:
>aliasQualifier [](http://example.com/codeflow?start=178&length=14)

Is this a part of ```explicitInterfaceSpecifier```? Then it can be calculated when needed.